src/validation.cpp

```diff
@@ -4347,7 +4347,7 @@ bool CChainState::LoadGenesisBlock(const CChainParams& chainparams)
         return true;
 
     try {
-        CBlock &block = const_cast<CBlock&>(chainparams.GenesisBlock());
+        const CBlock &block = chainparams.GenesisBlock();
```

Comment:
nit, `const CBlock& block`.

Reply:
Fixed the pointer alignment according to `.clang-format`.